### DISCUSSION TITLE
Add Category for each items   ///Item::Category

### DISCUSSION THREAD

**Discussion by ghost** [community/community#14770] [Other Feature Feedback, Questions, & Ideas]
> Please add category field for items

![FeedBack:: Category Pixel PNG](https://user-images.githubusercontent.com/101324558/162939016-a88509f6-d74e-48f2-a03b-d607858b657b.png)

**Thanks**

**Comment by queenofcorgis** [comment]
Hi there, would you mind on elaborating on this request please so we can better understand it?
